orm/hibernate-orm-6/pom.xml

```diff
@@ -51,7 +61,6 @@
 			<plugin>
 				<groupId>org.hibernate.orm.tooling</groupId>
 				<artifactId>hibernate-enhance-maven-plugin</artifactId>
-				<version>${version.org.hibernate}</version>
```

Comment:
We'll see if the build passes, but... I thought BOMs didn't apply to Maven plugins?

Reply:
yeah 🙈 it didn't. I was looking only at the dependabot side https://github.com/marko-bekhta/hibernate-test-case-templates/pull/4